astropy/cosmology/funcs/comparison.py

```diff
@@ -36,41 +37,28 @@
 # UTILITIES
 
 
+_CANT_BROADCAST: tuple[type, ...] = (table.Row, table.Table)
+"""Things that cannot broadcast.
+
+Have to deal with things that do not broadcast well. e.g.
+`~astropy.table.Row` cannot be used in an array, even if ``dtype=object``
+and will raise a segfault when used in a `numpy.ufunc`.
```

Comment:
A `segfault` should not be allowed to go without an issue over at numpy!

Reply:
I'll try to figure out a MWE.